sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/directconnectivity/ConsistencyWriterTest.java

```diff
@@ -24,11 +24,7 @@
 import reactor.core.publisher.DirectProcessor;
 import reactor.core.publisher.Mono;
 
-import java.util.AbstractMap;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
code style, no wildcard import:
https://peterdev.pl/2019/10/18/why-you-should-avoid-star-imports-in-intellij-idea/